test/utils/setupBabel.js

```diff
@@ -1,3 +1,4 @@
 require('@babel/register')({
   extensions: ['.js', '.ts', '.tsx'],
+  ignore: [/node_modules\/(?!@mui\/monorepo)/],
```

Comment:
We had two babel files with almost the same config.
I removed the one in the old `setup.js` file and used this one everywhere.